app/styles/nypr-ui/_colors.scss

```diff
@@ -26,3 +26,43 @@ $graywhite: #f8f8f8;            // rgb(248, 248, 248)
 // Social
 $facebook: #3b5998;             // rgb(59, 89, 152)
 $facebook-hover: #2a457c;
+
+// Image Filters
+$filter: rgba(0, 0, 0, .3);
```

Comment:
Yeah, I was going to keep it pretty unspecific until we added more gradients. But you're right, since I may not be the one adding the next one, might as well set a good naming convention now!